### PR TITLE
Add entities wrapper docs

### PR DESCRIPTION
**Pull request by mohabusama**
Entities wrapper docs

### CODE REVIEW

File: docs/user/check-ref/entities_wrapper.rst [hunk 1]
```diff
@@ -0,0 +1,84 @@
+Entities
+--------
+
+Provides access to ZMON entities.
+
+.. py:function:: entities(service_url, infrastructure_account, verify=True, oauth2=False)
```
Comment (hjacobs) [inline docs/user/check-ref/entities_wrapper.rst:6]: it should be noted that all the args are optional (pre-filled by the context)

File: docs/user/check-ref/entities_wrapper.rst [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+
+Methods of Entities
+^^^^^^^^^^^^^^^^^^^
+
+.. py:function:: search_local(**kwargs)
+
+    ::
+
+    Search entities in local infrastructure account. If `infrastructure_account` is not supplied in kwargs, then the one supplied upon intialization will be used.
```
Comment (Jan-M) [inline docs/user/check-ref/entities_wrapper.rst]: Rephrase for "initialization".

Search entities "local" to your filtered entities by using the same infrastructure_account as a default filter.

**Comment by hjacobs** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Pull request merged**